frontend/language/src/nb.json

```
@@ -491,14 +491,14 @@
   "local_changes.modal_loading_delete_local_changes": "Sletter lokale endringer",
   "local_changes_modal.download_all_button": "Last ned zip-fil av hele repoet",
   "local_changes_modal.download_all_files_description": "Du kan laste ned hele repoet, inkludert alle filene du har gjort endringer på fra din lokale maskin.",
-  "merge_conflict.body1": "Det finnes endringer i denne applikasjonen som er i konflikt med dine siste endringer.",
-  "merge_conflict.body2": "Måten å løse konflikten på er å fjerne dine siste endringer. Dersom du vil finne tilbake til endringene du nå mister, kan du laste ned siste versjon som zip-fil.",
+  "merge_conflict.body1": "Noen andre har endret appen på samme sted som deg. Hvis Del endringer ikke fungerer, kan du laste ned en zip-fil med endringene dine.",
+  "merge_conflict.body2": "Velg Slett mine endringer for å løse konflikten.",
```

Denne `<Trans>`-komponenten er litt snodig, men hvis jeg forstår det riktig, bør det ikke se slik ut for å få med `<strong>`-komponenten? Hvordan ser det ut når du tester det lokalt?
```suggestion
  "merge_conflict.body1": "Noen andre har endret appen på samme sted som deg. Hvis <1>Del endringer</1> ikke fungerer, kan du laste ned en zip-fil med endringene dine.",
  "merge_conflict.body2": "Velg <1>Slett mine endringer</1> for å løse konflikten.",
```

---

Litt usikker på hva du mener med Trans komponenten. Om du ser i kommentaren over så er det slik det ser ut 😄 
Trans er en fin måte å manipulere verdiene i teksten direkte, [står mer i docsene her](https://react.i18next.com/latest/trans-component) 😄

---

Det var dette med `<1>` jeg mente ja, bra.